test/producePublicFakeFactors.py

```diff
@@ -3,170 +3,156 @@
 import os
 
 ## Meta-data
-version = '20160809'
-tag     = 'v0.1.3'
+version = '20160913'
+channel= 'et'
```

Comment:
Probably better to implement a loop on all final states and categories, in order to produce the fake factors all together.